whois-client/src/main/java/net/ripe/db/whois/api/rest/domain/InfoMessage.java

```diff
@@ -0,0 +1,17 @@
+package net.ripe.db.whois.api.rest.domain;
+
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlRootElement;
+import net.ripe.db.whois.common.Message;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "infoMessage")
+public class InfoMessage extends WhoisResponseMessage {
+
+    public InfoMessage(){super();}
```

Comment:
Re-format so `super();` is on a separate line. This constructor should not be needed usually but I think it's required for deserialisable classes

Reply:
exactly. Is is used for deserialisation

Reply:
OK. So keep the constructor but reformat it please for consistency.

Reply:
Done